lib/sla.js

```diff
@@ -61,35 +93,94 @@ function addChannelsToBridge(channel, bridge) {
  * Creates a channel to a specified endpoint and places it in the Stasis app.
  * @param {Object} client - Object that contains information from the ARI
  *   connection.
+ * @param {Object} inbound - Channel that originally dialed into Stasis.
  * @param {Object} bridge - Object that defines the bridge to be passed to
  *   addChannelsToBridge().
  * @return {Q} Q promise object.
  */
-function createChannel(client, bridge) {
+function originateChannel(client, inbound, bridge) {
   var defer = Q.defer();
-  var channel = client.Channel();
-  var originate = Q.denodeify(channel.originate.bind(channel));
-  originate({endpoint: 'SIP/phone', app: 'sla'})
-    .catch(function (err) {
-      defer.reject(err);
+  var dialed = [client.Channel(), client.Channel()];
+  var originate = [];
+  var numToHangup = dialed.length;
+  var station = ['SIP/phone1', 'SIP/phone2'];
+  dialed.forEach(function (channel, i, dialed) {
```

Comment:
i is not very descriptive. At the very least call it index.

This would be simpler if you stored an array of objects instead that each contain a channel, station, et cetera. You can then just iterate over the objects and use object.channle or object.station instead of having to use an index into an array.